sphinx/texinputs/Makefile_t

```diff
@@ -12,29 +19,52 @@ LATEXOPTS =
 # format: pdf or dvi
 FMT = pdf
 
-LATEX = latex
-PDFLATEX = {{ latex_engine }}
-MAKEINDEX = makeindex
+{% if latex_engine == 'platex' -%}
+# latexmkrc is read then overridden by latexmkjarc
+LATEX = latexmk -r latexmkjarc -dvi
+PDFLATEX = latexmk -r latexmkjarc -pdfdvi -dvi- -ps-
+{% elif latex_engine == 'pdflatex' -%}
+LATEX = latexmk -dvi
+PDFLATEX = latexmk -pdf -dvi- -ps-
+{% elif latex_engine == 'lualatex' -%}
+LATEX = latexmk -lualatex
+PDFLATEX = latexmk -pdflua -dvi- -ps-
+{% elif latex_engine == 'xelatex' -%}
+LATEX = latexmk -pdfxe -dvi- -ps-
+PDFLATEX = $(LATEX)
+{% endif %}
+
+%.png %.gif %.jpg %.jpeg: FORCE_MAKE
+	extractbb '$@'
+
+{% if latex_engine == 'platex' -%}
+%.dvi: %.tex $(ALLIMGS) FORCE_MAKE
+	for f in *.pdf; do extractbb "$$f"; done
```

Comment:
I have added ``"`` quotes. The other image types are already quoted for spaces in recipe. But I know Sphinx docs says somewhere filenames with spaces are not supported for images. Nevertheless adding the ``"`` for symmetry reasons.